examples/hpc-enterprise-slurm.yaml

```diff
@@ -0,0 +1,233 @@
+        family: $(vars.family)
+        project: $(vars.project)
+
+  - id: n2_partition
+    source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    use: [n2_node_group, network1, homefs, projectsfs, scratchfs]
+    settings:
+      partition_name: n2
+      exclusive: false # allows nodes to stay up after jobs are done
+      enable_placement: false # the default is: true
+      is_default: true
+
+  - id: c2_node_group
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      node_count_dynamic_max: 20
+      machine_type: c2-standard-60 # this is the default
+      instance_image:
+        family: $(vars.family)
+        project: $(vars.project)
+      bandwidth_tier: "tier_1_enabled"
+      disk_type: pd-ssd
+      disk_size_gb: 100
+
+  # use `-p c2` to submit jobs to this partition:
+  # ex: `srun -p c2 -N 1 hostname`
+  - id: c2_partition
+    source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    use: [c2_node_group, network1, homefs, projectsfs, scratchfs]
+    settings:
+      partition_name: c2
+
+  - id: c2d_node_group
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      node_count_dynamic_max: 20
+      machine_type: c2d-standard-112
+      instance_image:
+        family: $(vars.family)
+        project: $(vars.project)
+      bandwidth_tier: "tier_1_enabled"
+      disk_type: pd-ssd
+      disk_size_gb: 100
+
+  - id: c2d_partition
+    source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    use: [c2d_node_group, network1, homefs, projectsfs, scratchfs]
+    settings:
+      partition_name: c2d
+
+  - id: c3_node_group
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      node_count_dynamic_max: 20
+      machine_type: c3-highcpu-176
+      instance_image:
+        family: $(vars.family)
+        project: $(vars.project)
+      bandwidth_tier: "tier_1_enabled"
+      disk_type: pd-ssd
+      disk_size_gb: 100
+
+  - id: c3_partition
+    source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    use: [c3_node_group, network1, homefs, projectsfs, scratchfs]
+    settings:
+      partition_name: c3
+
+  - id: a2_8_node_group
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      node_count_dynamic_max: 16
+      machine_type: "a2-ultragpu-8g"
+      bandwidth_tier: "gvnic_enabled"
+      instance_image:
+        family: $(vars.family)
+        project: $(vars.project)
+      disk_type: pd-ssd
+      disk_size_gb: 100
+
+  # use `-p a208` to submit jobs to this partition:
+  # ex: `srun -p a208 --gpus-per-node=8 -N 1 nvidia-smi`
+  - id: a2_8_partition
+    source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+    use: [a2_8_node_group, network1, homefs, projectsfs, scratchfs]
+    settings:
+      partition_name: a208
+      exclusive: true # this must be true if enable_placement is true
+      enable_placement: true
```

Comment:
maybe mention that these are defaults. It feels odd they are here and nowhere else.

Reply:
good point, I moved it to c2, which is the first partition after n2 and mention they are defaults, it is nice to show how they are set.